.github/workflows/production-heartbeat.yml

```diff
@@ -10,139 +10,16 @@ on:
     - cron: '45 13,17,21 * * 1,2,3,4,5'
 jobs:
   production-heartbeat:
-    strategy:
-      # By default, if any job in a matrix fails, all other jobs are immediately cancelled. This makes the jobs run to completion instead.
-      fail-fast: false
-      matrix:
-        os: [{vm: ubuntu-latest, exe: .sh}, {vm: windows-2019, exe: .cmd}]
-        node: ['lts/*']
-    runs-on: ${{ matrix.os.vm }}
-    timeout-minutes: 60
+    runs-on: macos-latest
```

Comment:
The old version of this file was just a copy of the one from v4 that I believe won't work properly anyway.
So I'm just directly calling over to the v4 job, which is why I pulled the internals of that job into their own flow on the v4 side.